app/assets/javascripts/discourse/tests/unit/utils/decorators-test.js

```diff
@@ -175,4 +177,78 @@ module("Unit | Utils | decorators", function (hooks) {
 
     assert.strictEqual(stub.otherCounter, 1);
   });
+
+  test("@observes works via .extends and native class syntax", async function (assert) {
```

Comment:
`.extends` -> `.extend` (that's the name of the Ember function, `extends` is the native class keyword 😅)

Reply:
Right 🤦 Fixed

Flake is unrelated to this PR - it's happening on `main` as well. Digging now 👀